data/strings/strings.txt

```diff
@@ -21785,6 +21743,48 @@
     zh-Hans = 加载地图
     zh-Hant = 載入地圖
 
+  [notification_channel_navigation]
```

Comment:
It already exists in this file:
```
  [prefs_group_route]
    tags = android,ios
    en = Navigation
    af = Navigasie
    ar = الملاحة
    be = Навігацыя
    bg = Навигация
```

Reply:
There's no need to duplicate the translation if the meaning is _exactly_ the same. You can rename the key to [navigation] and reuse it where necessary.

Reply:
I will re-use existing string, no problem.